specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/virtualNetwork.json

```diff
@@ -1496,11 +1500,36 @@
         },
         "privateEndpointNetworkPolicies": {
           "type": "string",
-          "description": "Enable or Disable apply network policies on private end point in the subnet."
+          "default": "Enabled",
+          "description": "Enable or Disable apply network policies on private end point in the subnet.",
+          "enum": [ 
+            "Enabled", 
+            "Disabled" 
+          ], 
+          "x-ms-enum": { 
+            "name": "PrivateEndpointNetworkPolicies", 
```

Comment:
Please add Prefix for enum name, For example "VirtualNetworkPrivateEndpointNetworkPolicies"